coalib/settings/Setting.py

```diff
@@ -44,6 +45,21 @@ def glob_list(obj, *args, **kwargs):
     return obj.__glob_list__(*args, **kwargs)
 
 
+def language_obj(name):
+    """
+    Convert a string into ``Language`` object.
+
+    :param name:        String containing language name.
+    :return:            ``Language`` object.
+    :raises ValueError: If the ``name`` contain invalid language name.
+    """
+    try:
+        return Language[name]
+    except AttributeError:
```

Comment:
This line of your patch is correct, but will need to change if https://github.com/coala/coala/pull/4643 is merged, as that fixes a fairly fundamental exception problem.